debian/changelog

```diff
@@ -1,3 +1,9 @@
+cloud-init (22.4.2-0ubuntu1) lunar; urgency=medium
+
+  * Upstream snapshot based on upstream commit 893df0d6.
+
```

Comment:
@aciba90 this also needs to include any LP bugs fixed 
Something like 
```
- Bugs fixed in this snapshot: (LP: #1997559)
```

Reply:
Amended, thanks.